[2025-02-18T13:48:20.865-0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T13:48:20.894-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: FirstDAG.Firsttask scheduled__2025-02-18T01:00:00+00:00 [queued]>
[2025-02-18T13:48:20.911-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: FirstDAG.Firsttask scheduled__2025-02-18T01:00:00+00:00 [queued]>
[2025-02-18T13:48:20.912-0600] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-18T13:48:20.944-0600] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): Firsttask> on 2025-02-18 01:00:00+00:00
[2025-02-18T13:48:20.952-0600] {standard_task_runner.py:72} INFO - Started process 93791 to run task
[2025-02-18T13:48:20.959-0600] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'FirstDAG', 'Firsttask', 'scheduled__2025-02-18T01:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmppwraiytn']
[2025-02-18T13:48:20.963-0600] {standard_task_runner.py:105} INFO - Job 25: Subtask Firsttask
[2025-02-18T13:48:21.030-0600] {task_command.py:467} INFO - Running <TaskInstance: FirstDAG.Firsttask scheduled__2025-02-18T01:00:00+00:00 [running]> on host IMNOTROY.
[2025-02-18T13:48:21.212-0600] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='FirstDAG' AIRFLOW_CTX_TASK_ID='Firsttask' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-18T01:00:00+00:00'
[2025-02-18T13:48:21.214-0600] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T13:48:21.218-0600] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T13:48:21.234-0600] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T13:48:21.235-0600] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=FirstDAG, task_id=Firsttask, run_id=scheduled__2025-02-18T01:00:00+00:00, execution_date=20250218T010000, start_date=20250218T194820, end_date=20250218T194821
[2025-02-18T13:48:21.297-0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T13:48:21.317-0600] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T13:48:21.319-0600] {local_task_job_runner.py:245} INFO - ::endgroup::
